### PR TITLE
fix: close modal on view creation

### PR DESCRIPTION
**Pull request by aaron-dernley**
## How does this PR change the system?

- Creating a view closes the modal when clicking the Create button
- Creating a view closes the modal when hitting Enter after entering a value
- Editing a view name pulls in the name of the view to the input

#### Video:

https://jam.dev/c/93a78fec-3a94-4af1-9b2a-133e1105dee1

## How was it tested?

- [X] Integration tests pass
- [X] Manual test: new functionality works in UI
- [X] Manual test: (regression check) creating a component still works

### CODE REVIEW

File: app/web/src/newhotness/AddViewModal.vue [hunk 1]
```diff
@@ -8,43 +8,45 @@
...
-      </nameForm.Field>
-      <Icon
-        v-if="wForm.bifrosting.value"
-        class="absolute right-2xs"
-        name="loader"
-        size="sm"
-        tone="action"
-      />
-    </label>
+    <form @submit.prevent="nameForm.handleSubmit()">
```
Comment (aaron-dernley) [inline app/web/src/newhotness/AddViewModal.vue:11]: I was unsure about this being the right way to handle this, however it does work..

Comment (wendybujalski) [reply app/web/src/newhotness/AddViewModal.vue:11]: This should capture both pressing enter or clicking the submission button. Let me take this for a test run to check for edge case issues!

File: app/web/src/newhotness/AddViewModal.vue
Comment (wendybujalski) [inline app/web/src/newhotness/AddViewModal.vue:1]: Took this PR for a spin, it looks like the AddViewModal also has other bugs which we should fix in this PR.

**Review by wendybujalski — CHANGES_REQUESTED** [review]

**Review by wendybujalski — APPROVED** [review]
